browser/brave_content_browser_client.cc

```diff
@@ -156,11 +154,9 @@ void BraveContentBrowserClient::BrowserURLHandlerCreated(
                           &webtorrent::HandleTorrentURLReverseRewrite);
 #endif
 #if BUILDFLAG(IPFS_ENABLED)
-  if (base::FeatureList::IsEnabled(ipfs::features::kIpfsFeature)) {
-    handler->AddHandlerPair(
-        &ipfs::ContentBrowserClientHelper::HandleIPFSURLRewrite,
-        &ipfs::ContentBrowserClientHelper::HandleIPFSURLReverseRewrite);
-  }
+  handler->AddHandlerPair(
```

Comment:
condition checks are done in those static functions content_browser_client_helper.cc now, remove the condition checks in this file.